### PR TITLE
Add Git Hooks

### PR DESCRIPTION
**Pull request by aitemr**
https://github.com/aitemr/awesome-git-hooks#readme

**[Explain what this list is about and why it should be included here]**

A curated list of awesome Git Hooks. Git hooks are scripts that run automatically every time a particular event occurs in a Git repository.

PR reviews:
#1489
#1487 

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

Continued from https://github.com/sindresorhus/awesome/pull/1467

### CODE REVIEW

**Comment by sindresorhus** [comment]
In the `Useful Git Hooks scripts` section, each hook should have a description.

---

The `Inspired by the awesome list thing.` sentence should be removed. You already have the Awesome badge.

---

The description starts with an uppercase character and **ends with a period**.

---

`occurs in a git repository.` => `occurs in a Git repository.`

**Comment by sindresorhus** [comment]
The list is a little bit thin on resources. There's a lot of good stuff around Git Hooks out there. I would recommend spending some more time on the list.

File: readme.md [hunk 1]
```diff
@@ -380,6 +380,7 @@
 - [PowerShell](https://github.com/janikvonrotz/awesome-powershell#readme) - Cross-platform object-oriented shell.
 - [Alfred Workflows](https://github.com/derimagia/awesome-alfred-workflows#readme) - Productivity app for macOS.
 - [Terminals Are Sexy](https://github.com/k4m4/terminals-are-sexy#readme)
+- [Git Hooks](https://github.com/aitemr/awesome-git-hooks#readme)
```
Comment (sindresorhus) [inline readme.md]: > The entry in the Awesome list should:
Include a short description about the project/theme of the list. It should not describe the list itself.
Example: - [Fish](…) - User-friendly shell., not - [Fish](…) - Resources for Fish..

**Comment by aitemr** [comment]
Hi, @sindresorhus!

Resolved issues 

> In the `Useful Git Hooks scripts` section, each hook should have a description.
> 
> The `Inspired by the awesome list thing.` sentence should be removed. You already have the Awesome badge.
> 
> The description starts with an uppercase character and **ends with a period**.
> 
> `occurs in a git repository.` => `occurs in a Git repository.`

Hi, @sindresorhus!

Thanks for your review. Fixed all comments

**Comment by aitemr** [comment]
> Has contribution guidelines.
> The file should be named contributing.md. Casing is up to you.

renamed

**Comment by catalinmiron** [comment]
Thanks for contributing @aitemr 

Take a quick look:

I would suggest you create a LICENSE file to officially make a CC0 repository. Take a look at this repository as an example (https://github.com/victorshinya/awesome-ibmcloud/blob/master/LICENSE).

**Comment by aitemr** [comment]
Hi, @catalinmiron!

Thanks for your review. [Added](https://github.com/aitemr/awesome-git-hooks/commit/0a6deae0a0ac71257b9710324a84608997e352b1)

**Comment by sindresorhus** [comment]
Please re-read my comment here: https://github.com/sindresorhus/awesome/pull/1498#discussion_r248992936 What you added is not correct.

**Comment by sindresorhus** [comment]
> The list is a little bit thin on resources. There's a lot of good stuff around Git Hooks out there. I would recommend spending some more time on the list.

This includes more tools and articles.

**Comment by sindresorhus** [comment]
Ping

**Comment by sindresorhus** [comment]
```
❯ awesome-lint https://github.com/aitemr/awesome-git-hooks
✖ Linting

  README.md:1:1
  ✖   1:1   Missing Awesome badge after the main heading                    remark-lint:awesome/badge
  ✖   1:1   Missing file contributing.md                                    remark-lint:awesome/contributing
  ✖   1:1   Missing newline character at end of file                        remark-lint:final-newline
  ✖  13:1   Table of Contents must be the first section                     remark-lint:awesome/toc
  ✖  16:1   ToC item "Tools" does not match corresponding heading "Soon 🙈"  remark-lint:awesome/toc
  ✖  35:3   Invalid list item link URL                                      remark-lint:awesome/list-item
  ✖  36:3   Invalid list item link URL                                      remark-lint:awesome/list-item
  ✖  37:3   Invalid list item link URL                                      remark-lint:awesome/list-item
  ✖  38:3   Invalid list item link URL                                      remark-lint:awesome/list-item
  ✖  39:3   Invalid list item link URL                                      remark-lint:awesome/list-item
  ✖  40:3   Invalid list item link URL                                      remark-lint:awesome/list-item
  ✖  41:3   Invalid list item link URL                                      remark-lint:awesome/list-item
  ✖  45:45  List item description must end with proper punctuation          remark-lint:awesome/list-item
  ✖  46:47  List item description must end with proper punctuation          remark-lint:awesome/list-item
  ✖  47:50  List item description must end with proper punctuation          remark-lint:awesome/list-item
  ✖  48:52  List item description must end with proper punctuation          remark-lint:awesome/list-item
  ✖  49:55  List item description must end with proper punctuation          remark-lint:awesome/list-item

  17 errors
```

https://github.com/sindresorhus/awesome-lint

File: readme.md [hunk 2]
```diff
@@ -380,6 +380,7 @@
 - [PowerShell](https://github.com/janikvonrotz/awesome-powershell#readme) - Cross-platform object-oriented shell.
 - [Alfred Workflows](https://github.com/derimagia/awesome-alfred-workflows#readme) - Productivity app for macOS.
 - [Terminals Are Sexy](https://github.com/k4m4/terminals-are-sexy#readme)
+- [Git Hooks](https://github.com/aitemr/awesome-git-hooks#readme) - A curated list of awesome Git Hooks.
```
Comment (sindresorhus) [inline readme.md]: > The entry in the Awesome list should:
> Include a short description about the project/theme of the list. **It should not describe the list itself.**

**Comment by aitemr** [comment]
Hi, @sindresorhus!

Sorry for the late response. I am fixing.

File: readme.md (hunk 2 above)
Comment (aitemr) [reply readme.md]: fixed

**Comment by aitemr** [comment]
Hi, sorry I completely missed the messages.
I just fixed all the above
I'd really appreciate it if you could take a look 🙏

**Comment by aitemr** [comment]
> Has contribution guidelines.
> The file should be named contributing.md. Casing is up to you.

Updated

**Comment by sindresorhus** [comment]
https://github.com/sindresorhus/awesome/pull/1498#discussion_r248992936 is not correct. Like my comment says, it should be about what Git hooks are, not what your list is about.

**Comment by sindresorhus** [comment]
> Includes a succinct description of the project/theme at the top of the readme. (Example)

**Comment by aitemr** [comment]
Good morning, @sindresorhus!

Updated.

File: readme.md [hunk 3]
```diff
@@ -380,7 +380,7 @@
 - [PowerShell](https://github.com/janikvonrotz/awesome-powershell#readme) - Cross-platform object-oriented shell.
 - [Alfred Workflows](https://github.com/derimagia/awesome-alfred-workflows#readme) - Productivity app for macOS.
 - [Terminals Are Sexy](https://github.com/k4m4/terminals-are-sexy#readme)
-
+- [Git Hooks](https://github.com/aitemr/awesome-git-hooks#readme) - are scripts that run automatically every time a particular event occurs in a Git repository.
```
Comment (sindresorhus) [inline readme.md]: The description should be independent from the title.

**Comment by sindresorhus** [comment]
Ping

**Pull request closed without merging**